### ISSUE TITLE
issue with aspect ratio

### ISSUE THREAD

**Issue by uuc110** [shubho0908/Certificate-Generator#3]
I saw the original one i have and the size of this certificate seems there is issue with sizing?

**Comment by shubho0908** [comment]
Might be, this project is about 1 year old, and haven't made many changes since then. They might have tweaked the aspect ratio of the certificates in between.

**Comment by shubho0908** [comment]
> I saw the original one i have and the size of this certificate seems there is issue with sizing?


Might be, this project is about 1 year old, and haven't made many changes since then. They might have tweaked the aspect ratio of the certificates in between.

**Comment by uuc110** [comment]
Would it be fine if I collaborate and fix it? if you can help me with little bit. I changed some fonts which look more similar and changed aspect ratio. just need few tweaking with sizing, you can look into it

**Comment by shubho0908** [comment]
> Would it be fine if I collaborate and fix it? if you can help me with little bit. I changed some fonts which look more similar and changed aspect ratio. just need few tweaking with sizing, you can look into it

Sure, I'd appreciate it.

**Comment by uuc110** [comment]
> > Would it be fine if I collaborate and fix it? if you can help me with little bit. I changed some fonts which look more similar and changed aspect ratio. just need few tweaking with sizing, you can look into it
> 
> Sure, I'd appreciate it.

Here I did some changes, Updates are here =>

1. Changed the Aspect ratio & gap between instructor and Name of user
**Before:**

![image](https://user-images.githubusercontent.com/44457947/233793761-8f9b83b3-ec8d-4aaa-a1fa-d3a51f7d9e19.png)

**After:**
![image](https://user-images.githubusercontent.com/44457947/233793583-e176595c-500e-427a-8c9e-59f0ad20e9a6.png)


2. Added "s" after instructor, was missing in output produced certificate
3. Added Min and hours feature ⭐
**Before:**
![image](https://user-images.githubusercontent.com/44457947/233793526-1137d11b-b8e1-4241-b073-3656f215e7ef.png)
**After:**
![image](https://user-images.githubusercontent.com/44457947/233793534-3b26cee3-5814-425a-8d8f-94a70f0431c7.png)


New issue:

1. Cramped Download button, when Certificate title name is big. it get cramped
![image](https://user-images.githubusercontent.com/44457947/233793500-068d33a1-972d-4537-8863-72d9d03bc247.png)

**Comment by shubho0908** [comment]
> > > Would it be fine if I collaborate and fix it? if you can help me with little bit. I changed some fonts which look more similar and changed aspect ratio. just need few tweaking with sizing, you can look into it
> > 
> > 
> > Sure, I'd appreciate it.
> 
> Here I did some changes, Updates are here =>
> 
> 1. Changed the Aspect ratio & gap between instructor and Name of user
>    **Before:**
> 
> ![image](https://user-images.githubusercontent.com/44457947/233793761-8f9b83b3-ec8d-4aaa-a1fa-d3a51f7d9e19.png)
> 
> **After:** ![image](https://user-images.githubusercontent.com/44457947/233793583-e176595c-500e-427a-8c9e-59f0ad20e9a6.png)
> 
> 2. Added "s" after instructor, was missing in output produced certificate
> 3. Added Min and hours feature ⭐
>    **Before:**
>    ![image](https://user-images.githubusercontent.com/44457947/233793526-1137d11b-b8e1-4241-b073-3656f215e7ef.png)
>    **After:**
>    ![image](https://user-images.githubusercontent.com/44457947/233793534-3b26cee3-5814-425a-8d8f-94a70f0431c7.png)
> 
> New issue:
> 
> 1. Cramped Download button, when Certificate title name is big. it get cramped
>    ![image](https://user-images.githubusercontent.com/44457947/233793500-068d33a1-972d-4537-8863-72d9d03bc247.png)

Thanks, Mate! please make a request so that I can merge it.
